utils/CENNZTransaction.ts

```diff
@@ -2,6 +2,7 @@ import { Api, SubmittableResult } from "@cennznet/api";
 import Emittery from "emittery";
 import { CENNZ_EXPLORER_URL } from "@/constants";
 import { Event } from "@polkadot/types/interfaces";
```

Comment:
can remove `Event` and `Api` type imports

Reply:
good nit!